std/ndn/mgmt_2022/definitions.go

```diff
@@ -337,3 +338,19 @@ type CsQuery struct {
 	PacketSize      uint64
 	FreshnessPeriod uint64
 }
+
+type PrefixInjection struct {
+	//+field:sequence:enc.Wire:wire
```

Comment:
`wire` is for zero-copy. If you don't need it, you can use `[]byte` and `//+field:binary`, which is simpler to handle